src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,7 @@
-export const UserInfo = () => {};
+import { UserInfoType } from '../../types/UserInfoType';
+
+export const UserInfo: React.FC<UserInfoType> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
+);
```

Comment:
you can only call UserInfo if your user is not null, then you don't need ? after user
and also use destructuring, for the user